main.go

```diff
@@ -339,17 +64,17 @@ func main() {
 	if len(opts.FcBinary) != 0 {
 		finfo, err := os.Stat(opts.FcBinary)
 		if os.IsNotExist(err) {
-			log.Fatalf("Binary, %q, does not exist: %v", opts.FcBinary, err)
+			return fmt.Errorf("Binary, %q, does not exist: %v", opts.FcBinary, err)
```

Comment:
"Binary %q does not exist: %v"

I don't think you need the comma.  Same for the other error messages below.